pulumi/tools/chocolateyinstall.ps1

```diff
@@ -8,4 +8,17 @@ $packageArgs = @{
   file64         = "$toolsDir\pulumi-vX.X.X-windows-x64.zip"
 }
 
-Install-ChocolateyZipPackage  @packageArgs
\ No newline at end of file
+Install-ChocolateyZipPackage  @packageArgs
+
+$bindir = Join-Path -Path $toolsDir -ChildPath "bin"
+
+# create empty sidecar so shimgen ignores them.
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-python.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-nodejs.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-go.exe.ignore") -Value $null
+Set-Content -Path (Join-Path -Path $toolsDir -ChildPath "pulumi-language-dotnet.exe.ignore") -Value $null
+
+Install-ChocolateyPath $bindir 'Machine'
+
+refreshenv
```

Comment:
I believe this only works when users have the Chocolatey profile loaded. `Update-SessionEnvironment` is the powershell equivalent. 
```suggestion
Update-SessionEnvironment
```

Reply:
@stack72 since Chocolatey spawns its own PowerShell host, this line won't actually have any bearing on the shell that was used to perform the installation of the pulumi package.  As a result, after doing `choco install pulumi` the end user won't be able to run `pulumi` until they either restart the shell used to perform the installation, or manually run the `refreshev` or `Update-SessionEnvironment`.

I would suggest that an addition to the installation docs is made to include mention of this.